Cargo.toml

```diff
@@ -14,7 +14,7 @@ edition = "2018"
 libc = "0.2.62"
 
 [target.'cfg(windows)'.dependencies]
-winapi = { version = "0.3.5", features = ["handleapi", "namedpipeapi", "processthreadsapi", "winnt"] }
+windows-sys = { version = "0.42.0", features = ["Win32_Foundation", "Win32_System_Pipes", "Win32_Security", "Win32_System_Threading"] }
```

Comment:
Thanks! Looks like `winapi` hasn't been updated in a couple years, so this seems like a good change. Do you know if we any of these features can be removed, or are they all needed? (I don't have a Windows machine with me today to just check.)

Reply:
The features are taken from its documentation, for example https://docs.rs/windows-sys/latest/windows_sys/Win32/System/Pipes/fn.CreatePipe.html, all of them are required